build-tools/timing/timing.projitems

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <_TopDir>$(MSBuildThisFileDirectory)..\..\</_TopDir>
+  </PropertyGroup>
+  <ItemGroup>
+    <XamarinAndroidProject Include="$(_TopDir)samples\HelloWorld\HelloWorld.csproj">
```

Comment:
We should pick a more semantically meaningful name here than `@(XamarinAndroidProject)`. What *is* a "Xamarin Android Project" such that `src/Mono.Android`, `tests/locales/Xamarin.Android.Locale-Tests`, and many more *don't* apply?

I would suggest `@(XACaptureBuildTimingProject)`.